prueba/src/palindromo/PalindromeIdentifier.java

```diff
@@ -0,0 +1,25 @@
+package palindromo;
+
+public class PalindromeIdentifier {
+
+	public boolean palindromeDetect(String phrase) {
+
+		String palindrome = phrase.replace(" ", "");
+		
+		boolean itsPalindrome;
+		int last;
+		int countEqualLetters = 0;
+		for (int first = 0; first < palindrome.length() - 1; first++) {
```

Comment:
If you call this variable as first, it would be a lie, because only at the first phase of the `for`, it will be the first character, but as the for loop goes on, it will be the last at the end, so maybe you can also rename the variables with iterator names as i, j or k.